lib/services/local/desktops.go

```diff
@@ -38,8 +38,61 @@ func NewWindowsDesktopService(backend backend.Backend) *WindowsDesktopService {
 	return &WindowsDesktopService{Backend: backend}
 }
 
+func (s *WindowsDesktopService) getWindowsDesktop(ctx context.Context, name, hostID string) (types.WindowsDesktop, error) {
+	key := backend.ExactKey(windowsDesktopsPrefix, hostID, name)
+	item, err := s.Get(ctx, key)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	desktop, err := s.itemToWindowsDesktop(item)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	return desktop, nil
+}
+
+func (s *WindowsDesktopService) itemToWindowsDesktop(item *backend.Item) (types.WindowsDesktop, error) {
```

Comment:
style: make it clear that this function doesn't actually depend on the struct

```suggestion
func (*WindowsDesktopService) itemToWindowsDesktop(item *backend.Item) (types.WindowsDesktop, error) {
```